scripts/generate_openapi_spec.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# This file is part of REANA.
+# Copyright (C) 2017 CERN.
+#
+# REANA is free software; you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation; either version 2 of the License, or (at your option) any later
+# version.
+#
+# REANA is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+# A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with REANA; if not, see <http://www.gnu.org/licenses>.
+#
+# In applying this license, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization or
+# submit itself to any jurisdiction.
+
+"""OpenAPI specification file generator script."""
+
+import json
+import os
+
+import click
+from apispec import APISpec
+from flask import current_app
+from flask.cli import with_appcontext
+from swagger_spec_validator.validator20 import validate_json
+
+# Import your marshmallow schemas here
+# from example_package.schemas import Example_schema,
+
+# Software title. E.g. just name of the module exposing the API.
+__title__ = "REANA Server"
+
+# Short description of the API. Supports GitHub Flavored Markdown.
+__api_description__ = "Submit workflows to be run on REANA Cloud"
+
+# Version of the API provides, not version of the OpenAPI specification.
+__api_version__ = "0.1"
+
+# Filepath where the OpenAPI specification file should be written to.
+__output_path__ = "temp_openapi.json"
+
+
+@click.command()
+@click.option('-t', '--title',
+              help='Title of package/component exposing the API')
```

Comment:
If people will copy/paste this code, as it were, then perhaps `help=__title__` directly?